cmd/podman/podman.go

```diff
@@ -0,0 +1,16 @@
+// +build linux
+
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"os"
+)
+
+func CheckForRegistries() {
+	if _, err := os.Stat("/etc/containers/registries.conf"); err != nil {
+		if os.IsNotExist(err) {
+			logrus.Warn("unable to find /etc/containers/registries.conf. some podman (image shortnames) commands may be limited")
```

Comment:
What about this is linux-specific? This seems like it ought to build on OS X fine

Reply:
there is no use for registries.conf on the remote client side.  all of that is handled on the server side.

Reply:
Swapping to a LocalRuntime method does sound like a better approach